merlin/config/utils.py

```diff
@@ -81,4 +81,4 @@ def get_priority(priority: Priority) -> int:
         raise ValueError(f"Invalid priority: {priority}")
 
     priority_map = determine_priority_map(CONFIG.broker.name.lower())
-    return priority_map.get(priority, 5)  # Default to 5 for MID or unknown priorities
+    return priority_map.get(priority, 5)  # Default to 5 for unknown priorities
```

Comment:
Might as well use it `return priority_map.get(priority, MID)`

Reply:
Since `Priority.MID` is an enum then I can't return it here since that will crash Celery.

Reply:
Yes, I changed it to the result of the enum selection.